packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php

```diff
@@ -91,11 +91,16 @@ public function uploadImage(array $data, ThemeCustomization $theme)
                 if (($data['type'] ?? '') == 'static_content') {
                     return Storage::url($path);
                 }
-
-                $options['images'][] = [
+                $data = [
                     'image' => 'storage/'.$path,
-                    'link'  => $image['link'],
+                    'link'  => $image['link']
                 ];
+
+                if(isset($image['title'])) {
+                    $data['title']  = $image['title'];
+                }
```

Comment:
Hi, 
There is no need to perform an extra check. You just need to use the code below.

```php 
$options['images'][] = [
   'image' => 'storage/'.$path,
   'link'  => $image['link'],
   'title' => $image['title'],
];

Reply:
Use the below code snippet.
```php
$options['images'][] = [
   'image' => 'storage/'.$path,
   'link'  => $image['link'],
   'title' => $image['title'],
];

```
After that execute the command into the root of the project.
```shell
vendor\bin\pint
```